firmware/applications/SignBuddy/Makefile

```diff
@@ -81,6 +81,11 @@ CHIPSET_DEF = STM32L476xx
 BOARD_DEF = BOARD_DISCO
 endif
 
+ifeq ($(BOARD_VERSION), NUCLEO)
```

Comment:
We could modify the makefile to have `make nucleo` and `make discovery`, which sets the environment variable BOARD_VERSION and then calls build

Reply:
yeah this is ideal